packages/ui/src/components/Menu/index.tsx

```diff
@@ -1,43 +1,36 @@
-import { Menu as HeadlessMenu } from '@headlessui/react';
+import * as RadixMenu from '@radix-ui/react-dropdown-menu';
 import { useRouter } from 'next/router';
 import { MouseEventHandler, ReactElement, ReactNode } from 'react';
 
 type MenuProps = {
   children: ReactNode;
 };
 
-export const Menu = ({ children, ...props }: MenuProps) => {
```

Comment:
I removed the `props` object because unless we add types to `MenuProps`, it's kinda useless.